VSCODE.md

```diff
@@ -0,0 +1,28 @@
+
+### vscode in-container development
+
+Main doc: https://code.visualstudio.com/docs/remote/containers
+
+#### Requirements
+
+- Docker
+- VSCode https://code.visualstudio.com/
+   - with remote development extension pack: https://aka.ms/vscode-remote/download/extension
+- (optional) add your github ssh key to agent if you want to be able to access non-public repos
+```sh
+ssh-add ~/.ssh/id_rsa # or your ssh key for github
+```
+
+#### Setup
+
+- start vscode and open repository
```

Comment:
Perhaps we should add a note here to modify `.devcontainer.json` to configure your container.